test/generic.js

```diff
@@ -439,7 +439,7 @@ describe('Generic headers', () => {
             url: '/'
         });
 
-        expect(res.statusCode).to.equal(200);
+        expect(res.statusCode).to.equal(204);
```

Comment:
`return ''` on line 433 is being treated as an empty response and is a 204 (no content) rather than a 200 like before.